nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/json/JsonTreeReader.java

```diff
@@ -66,6 +67,17 @@
     private volatile String dateFormat;
     private volatile String timeFormat;
     private volatile String timestampFormat;
+    private volatile String skipToNestedJsonField;
+
+    public static final PropertyDescriptor SKIP_TO_NESTED_JSON_FIELD = new PropertyDescriptor.Builder()
+            .name("skip-to-nested-json-field")
+            .displayName("Skip To Nested JSON Field")
```

Comment:
Recommend adjusting the wording to remove `JSON`, since the component already includes `Json` in the class name.
```suggestion
            .name("starting-field-name")
            .displayName("Starting Field Name")
```